dist/stat_data.py

```diff
@@ -210,9 +214,19 @@ def __init__(self, name, desc, flags=''):
     CacheStat('cache_pages_inuse',
         'pages currently held in the cache', 'no_clear,no_scale'),
     CacheStat('cache_read', 'pages read into cache'),
+    CacheStat('cache_read_lookaside',
+        'pages read into cache requiring lookaside entries'),
```

Comment:
The stats are looking better, but I'm still confused by how many there are and why they are spread out into different grouping.  I saw the count of inserts and bytes inserted and assumed there would be a count of reads and bytes read.  But that's not what is there (maybe because we don't track those in the data source stats?).

Instead, there are is a `rec_*` stat for page writes and a `cache_*` stat for page reads that use the lookaside table.  Plus there are `cache_lookaside_*` stats for operations on the lookaside table.

I'd like to simplify the stats, if possible, so we know what to look for to decide (1) whether updates are being written to the lookaside table, (2) whether reads are having to check it and (3) if it is growing very large.  Maybe just have `cache_read_lookaside` and rename `rec_pages_lookaside` to `cache_write_lookaside`?

If we need finer-grained information about the numbers of operations or bytes written, we could add them later?